AlphaWallet/Extensions/Data.swift

```diff
@@ -19,8 +19,7 @@ extension Data {
         return "0x" + self.hex()
     }
 
-    //NOTE: as minimum chunck is as min time it will be executed, during testing we found that optimal chunck size is 100, but seems it could be optimized more, execution time (0.2 seconds), pretty good and doesn't block UI
-    init(hex value: String, chunkSize: Int = 100) {
+    init(_hex value: String, chunkSize: Int) {
```

Comment:
Do we still need to replace `hex:` with `_hex:` in both initializers?

Reply:
i updated `hex:` with `_hex:` because there is existed implementation of .init(hex: ) in CryptoSwift that is using by default, app was using those implementation, even if we wanted to use our own

Reply:
Ah, ok. Hmm.. Could you add a comment to the init(_hex value:) to mention this? (why it's a _hex:). Merge at your convenience after that.